steps/comment-on-github-pull-request/0.9.1/step.yml

```diff
@@ -0,0 +1,60 @@
+title: Comment on GitHub Pull Request
+summary: |
+  Comment on [GitHub](https://github.com) Pull Request or Issue
+description: |
+  You can send a any comment on [GitHub](https://github.com) Pull Request or Issue.
+
+  To setup a **GitHub personal access token** visit: https://github.com/settings/tokens
+website: https://github.com/kvvzr/bitrise-step-comment-on-github-pull-request
+source_code_url: https://github.com/kvvzr/bitrise-step-comment-on-github-pull-request
+support_url: https://github.com/kvvzr/bitrise-step-comment-on-github-pull-request/issues
+published_at: 2018-11-05T12:14:40.961726867+09:00
+source:
+  git: https://github.com/kvvzr/bitrise-step-comment-on-github-pull-request.git
+  commit: e266acb4355d87ac35989dac88cd72967ab5e280
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+type_tags:
+- notification
+toolkit:
+  go:
+    package_name: github.com/kvvzr/bitrise-step-comment-on-github-pull-request
+deps:
+  brew:
+  - name: git
+  - name: wget
+  apt_get:
+  - name: git
+  - name: wget
+is_requires_admin_user: true
+is_always_run: false
```

Comment:
in case of `is_always_run: false` the step will not run if any previous step fails.
do not you want to send messages to PRs and issues in case of build failure?
All of our notification type steps has: `s_always_run: true`, to send messages in failed builds as well.